lib/puppet/transaction/report.rb

```diff
@@ -183,7 +194,7 @@ def initialize(kind, configuration_version=nil, environment=nil, transaction_uui
     @host = Puppet[:node_name_value]
     @time = Time.now
     @kind = kind
-    @report_format = 5
+    @report_format = 6
```

Comment:
> Where is the policy recorded for this thing incidentally? 

It isn't. This would be a great time to agree on where to record this policy. And to record it. :)